### DISCUSSION TITLE
Should we enable comments on documentation.eccenca.com?

### DISCUSSION THREAD

**Discussion by seebi** [eccenca/documentation.eccenca.com#21] [Polls]
I've enabled the [github discussion feature](https://github.com/eccenca/documentation.eccenca.com/discussions) for the documentation.eccenca.com repository as well as [a comments option with giscus](https://giscus.app) on the page (which uses the github backend for comments and discussion).

Currently this is enabled only for the cmemc pages.

Please test it on the here: https://dev.documentation.eccenca.com/automate/cmemc-command-line-interface/

Should we enable it per default (and disable it for some parts manually)?

**Comment by rpietzsch** [comment]
My take would be to enable it everywhere and see what happens...

**Reply by seebi** [reply]
At least, I do not want to have it on the home page ...

**Reply by rpietzsch** [reply]
👍
